templates/testpmd/README.md

```diff
@@ -95,9 +95,11 @@ Note that `containerDisk` value should be replaced with the actual repo where th
 
 
 ## Extra steps on the running VM
-DPDK requires that the driver bound to the interface that uses it (the SR-IOV interface in the VM) is vfio-pci.
+DPDK requires that the driver bound to the interface that uses it (the SR-IOV interface in the VM) is one of few models.
+If this driver model is iavf, then the VF interface (i.e. the SR-IOV NIC in the VM) should be actively bound to an vfio-pci driver by the user.
 It might be, however, that the driver supported by the node is different. In that case, the user would have to actively replace the default driver (e.g. iavf) which is bound to the SR-IOV NIC in the VM.
-You can achieve that by running the scripts/change-nic-driver.sh script in the VM (copy its contents to the VM).
+This can be achieved by running the scripts/change-nic-driver.sh script in the VM (copy its contents to the VM).
+If the driver in use is a Mellanox driver - the script will exit, and the user should refer to [https://doc.dpdk.org/guides/nics/mlx5.html#usage-example] (https://doc.dpdk.org/guides/nics/mlx5.html#usage-example).
```

Comment:
As far as I know changing the driver for Mellanox card VF's is the same as for Intel cards.
We used to have some nodes with Intel cards and some with Mellanox card on Kubevirt U/S CI cluster,
the logic for changing the driver was the same, [here is how it is done on kubevirtci](https://github.com/kubevirt/kubevirtci/blob/main/cluster-up/cluster/kind-1.22-sriov/sriov-node/configure_vfs.sh#L5-L30).